frontend/packages/schema/src/parser/tbls/parser.ts

```diff
@@ -329,6 +330,38 @@ function processTable(tblsTable: {
   ]
 }
 
+/**
+ * Process enums from tbls schema
+ */
+function processEnums(
+  tblsEnums:
+    | Array<{
+        name: string
+        values: string[]
+      }>
+    | undefined,
+): Enums {
+  const enums: Enums = {}
+
+  if (!tblsEnums) {
+    return enums
+  }
+
+  for (const tblsEnum of tblsEnums) {
+    const enumName = tblsEnum.name.includes('.')
+      ? tblsEnum.name.substring(tblsEnum.name.lastIndexOf('.') + 1)
+      : tblsEnum.name
+
+    enums[enumName] = anEnum({
+      name: enumName,
+      values: tblsEnum.values,
+      comment: null,
```

Comment:
tbls currently does not support comments for enums.
https://github.com/k1LoW/tbls/blob/174ce55ee29a0955e77071ddb076a050628b8b52/drivers/postgres/postgres.go#L525-L526